packages/gatsby-theme-newrelic/src/utils/related-resources/fetchRelatedResources.js

```diff
@@ -1,18 +1,15 @@
 const fs = require('fs');
 const search = require('./search');
 
-module.exports = async ({ node, siteUrl }, pluginOptions) => {
+module.exports = async ({ pathname, node, siteUrl }, pluginOptions) => {
```

Comment:
```suggestion
module.exports = async ({ pathname, node, siteUrl }, swiftypeOptions) => {
```

Could we rename this so that we don't confuse this with the theme options? This makes it a bit more clear that these aren't just the full set of options passed to the theme.